eventnotifier/slack/slack.go

```diff
@@ -110,12 +111,17 @@ func (n *Notifier) buildSlackBlocks(dis v1beta1.Disruption, notifType types.Noti
 }
 
 // Notify generates a notification for generic k8s events
-func (n *Notifier) Notify(dis v1beta1.Disruption, event corev1.Event, notifType types.NotificationType) error {
-	headerText := utils.BuildHeaderMessageFromDisruptionEvent(dis, notifType)
+func (n *Notifier) Notify(obj client.Object, event corev1.Event, notifType types.NotificationType) error {
+	dis, ok := obj.(*v1beta1.Disruption)
```

Comment:
Okay I'm now seeing we're explicitly only implementing http notifications for crons. why not slack too?

Reply:
Also in order to reduce the size of the PR. I can do it in another PR.